Build/Build.csproj

```diff
@@ -51,8 +51,8 @@
     <None Include="Remotion.Server.build" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Remotion.BuildScript, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Remotion.BuildScript.2.0.0-alpha018\lib\net45\Remotion.BuildScript.dll</HintPath>
```

Comment:
I think MK said something about this in his review? Just a reminder that you might have to fix it here too

Reply:
Indeed.

also, it's not just a revert, it's a verision fix that's required.